openshift/system/Dockerfile

```diff
@@ -58,6 +58,7 @@ RUN export ${BUNDLER_ENV} >/dev/null \
     && chmod g+w -vfR log tmp public/assets db/sphinx \
     && umask 0002 \
     && cd /opt/system \
+    && npm cache clean --force \
```

Comment:
Just changing the version of the cache key should be good. No need of this I think

Reply:
Again, this will be run inside the docker container. This cache is different from circleci's one. I'm not even sure there will be a cache inside the container.